canonical-websites/vanillaframework.io
# Add 404 / 500 pages

## Done

- Add 404 / 500 pages
- Fix title case of 'Vanilla framework' across the site

## QA

- Pull code
- Run `./run serve --watch`
- the only way I can see them locally is to copy/paste contents of 404.html and 500.html into the index.html page and view at: http://0.0.0.0:8014/

## Issue / Card

Fixes #54
